.github/workflows/reusable-test.yml

```diff
@@ -103,19 +103,23 @@ jobs:
           ## get package directory
           mkdir -p coverage
           yarn workspace ${{ matrix.package }} exec mv coverage/coverage-final.json ../../coverage/$PACKAGE_NAME.json
-      - uses: actions/upload-artifact@v3
+      - name: Sanitize package name
+        id: sanitize
+        run: echo "MATRIX_PACKAGE_NAME=${{ matrix.package }}" | sed 's/\//-/g' >> $GITHUB_ENV
```

Comment:
forward slash is not allowed, since the `matrix.package` is something like `@asap-hub/algolia` we were having these errors:
 
<img width="1030" alt="forward slash" src="https://github.com/user-attachments/assets/99489163-010a-4d31-9486-47d61bb5453a" />

so this part replaces `/` by `-`